python/rmm/_cuda/gpu.py

```diff
@@ -1,59 +1,36 @@
 # Copyright (c) 2020, NVIDIA CORPORATION.
 
-from rmm._cuda.gpu cimport (
-    CUresult,
-    cudaDeviceAttr,
-    cudaDeviceGetAttribute,
-    cudaDeviceProp,
-    cudaDriverGetVersion,
-    cudaError,
-    cudaError_t,
-    cudaGetDeviceCount,
-    cudaGetDeviceProperties,
-    cudaGetErrorName,
-    cudaGetErrorString,
-    cudaRuntimeGetVersion,
-    cuDeviceGetName,
-    cuGetErrorName,
-    cuGetErrorString,
-)
-
-from enum import IntEnum
+from cuda import cuda, cudart
 
 
 class CUDARuntimeError(RuntimeError):
-
-    def __init__(self, cudaError_t status):
+    def __init__(self, status: cuda.CUresult.CUDA_SUCCESS):
```

Comment:
Why is the type of `status` set as `cuda.CUresult.CUDA_SUCCESS`?

Reply:
Should be `cudart.cudaError_t`.
```suggestion
    def __init__(self, status: cudart.cudaError_t):
```

Reply:
Fixed